libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/ConfigService.java

```diff
@@ -38,7 +38,15 @@ public abstract class ConfigService {
 
     @Getter @Setter private static boolean local = false;
 
-    @Getter @Setter private List<String> featureSet;
+    private final ThreadLocal<List<String>> featureSet = new ThreadLocal<>();
+
+    public List<String> getFeatureSet() {
+        return featureSet.get();
+    }
+
+    public void setFeatureSet(List<String> featureSet) {
+        this.featureSet.set(featureSet);
+    }
```

Comment:
This is required so API tests running on locally-running core-back don't overlap featureSet between tests.